shared/test/test_files.rb

```diff
@@ -344,6 +344,38 @@ def test_bogus_metadata
     assert not_found?
   end
 
+  def test_rename_operations
+    filename = @api.randomize_filename('Mixed Case With Space.html')
+    escaped_filename = URI.escape(filename)
+    filename2 = @api.randomize_filename('Another Name With Spaces.html')
+    escaped_filename2 = URI.escape(filename2)
+    delete_all_file_versions(filename)
+    delete_all_file_versions(filename2)
```

Comment:
This can now be condensed to `delete_all_file_versions(filename, filename2)`.

Also, the other tests delete for the escaped filenames as well. Is that necessary as well?